bin/download_all_data.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
```

Comment:
Would it make sense to create a subworkflow that took care of all this downloads, this way if one of the downloads fails the others won't be affected. Also, probably will be more nextflow friendly. The only think is that then we will need to implement a parameter for each of the DBs or parameters, what do you think?

Reply:
It's a great idea and it can speed up the downloading process due to parallelisation. Actually we don't need a parameter for each DB (using the general param --skip_download and having a module for each DB or params). But we have to make sure that all of them succeeds before moving to the next step.